src/script_lua.c

```diff
@@ -1251,8 +1249,13 @@ void luaRegisterRedisAPI(lua_State* lua) {
 
     lua_pushstring(lua,"REPL_ALL");
     lua_pushnumber(lua,PROPAGATE_AOF|PROPAGATE_REPL);
+    lua_settable(lua,-3);
 
+    /* redis.acl_check_cmd_permissions */
+    lua_pushstring(lua,"acl_check_cmd_permissions");
```

Comment:
maybe that interface name is too verbose (ACL and permissions refer to the same thing).
i know the module API is named like that, but here it feels wrong(er).
how about a more generic `check_cmd_permissions` or `acl_check_cmd`

Reply:
+1 for `acl_check_cmd`